src/server/system_services/replication_store.js

```diff
@@ -102,6 +102,14 @@ class ReplicationStore {
 
         return reduced_replications;
     }
+
+    async find_aws_log_based_replication_rules() {
+        dbg.log0('find_aws_log_based_replication_rules: ');
+        const replications = await this._replicationconfigs.find({ deleted: null });
+
+        const reduced_replications = _.filter(replications, repl => repl.aws_log_replication_info);
+        dbg.log0('find_aws_log_based_replication_rules: ', reduced_replications);
```

Comment:
As we discussed - return reduced_replications and filter aws logs replication in the db query